skin-resque-frontend/src/components/Text/Text.scss

```diff
@@ -0,0 +1,21 @@
+@import url('https://fonts.googleapis.com/css2?family=Unna&display=swap');
+
+p {
+    font-family: 'Unna', serif;
+}
+
+.small {
```

Comment:
Perhaps specify class to signify that it's meant for text? I think something like text-small or text-s would work

Reply:
@dpionk what do you think about it?

Reply:
I think it is good as it is